articles/service-fabric/service-fabric-get-started-containers.md

```diff
@@ -327,6 +327,52 @@ NtTvlzhk11LIlae/5kjPv95r3lw6DHmV4kXLwiCNlcWPYIWBGIuspwyG+28EWSrHmN7Dt2WqEWqeNQ==
 </ServiceManifestImport>
 ```
 
+If there are multiple apps on a cluster with same credentials, you can configure cluster-wide credentials by adding a "UseDefaultRepositoryCredentials" attribute to ContainerHostPolicies in ApplicationManifest.xml with a "true/false" Boolean value.

```

Comment:
I think we should reorder the sections:

1) Explain the DefaultContainerRepository* settings and what they are used for.
2) Describe how an application can use the DefaultContainerRepository.

Also, this is more about multiple apps using a common repository, as opposed to same credentials (since you can have same credentials but different repos).